### PR TITLE
Reduce the number of vector pairs from 32 to 16.

### PR DESCRIPTION
**Pull request by chorman0773**

### CODE REVIEW

File: specs/Vector.md [hunk 1]
```diff
@@ -41,10 +42,10 @@ The notable changes are as follows: The third most significant bit (previously)
 
 ### Vector Register Operands
 
-A new class of operand, modifying the Register operand structure is defined for use in vectorized instructions. The structure of vector register operands is the following `[001 y ssss x 01 rrrrr]`
+A new class of operand, modifying the Register operand structure is defined for use in vectorized instructions. The structure of vector register operands is the following `[001 y ssss rrrrrrrr]`
 
 Where: `y` and `x` are reserved for future use, `ssss` is the extended size control value=`log2(size)`, and `1rrrrr` is the vector register number discarding the least significant bit.
```
Comment (rdrpenguin04) [inline specs/Vector.md:47]: What's `x`?

**Review by rdrpenguin04 — CHANGES_REQUESTED** [review]
Other than my one comment, this seems good!

File: specs/Vector.md (hunk 1 above)
Comment (chorman0773) [reply specs/Vector.md:47]: Oh yeah.

**Review by rdrpenguin04 — APPROVED** [review]
Seems Good

**Pull request merged**